Makefile

```diff
@@ -21,11 +21,14 @@ LIBSPEEXDSP_EXPORTS:='_speex_resampler_init','_speex_resampler_process_interleav
 RECORDER=$(OUTPUT_DIR)/recorder.min.js
 RECORDER_SRC=$(INPUT_DIR)/recorder.js
 
+INTERCEPTOR=$(OUTPUT_DIR)/interceptor.min.js
+INTERCEPTOR_SRC=$(INPUT_DIR)/interceptor.js
+
 WAVE_WORKER=$(OUTPUT_DIR)/waveWorker.min.js
 WAVE_WORKER_SRC=$(INPUT_DIR)/waveWorker.js
 
 
-default: $(LIBOPUS_ENCODER) $(LIBOPUS_DECODER) $(RECORDER) $(WAVE_WORKER) test
+default: $(LIBOPUS_ENCODER) $(LIBOPUS_DECODER) $(RECORDER) $(INTERCEPTOR) $(WAVE_WORKER) test
```

Comment:
Adding the interceptor.js does not seem necessary, as the changes have been made to `recorder.js`